Makefile.common

```diff
@@ -34,7 +34,7 @@ MD_CFLAGS+=-mno-red-zone
 endif
 
 # Likewise.
-LDFLAGS=-nostdlib -z max-page-size=0x1000 -static $(HOST_LDFLAGS)
+LDFLAGS=-nostdlib -z max-page-size=0x1000 $(HOST_LDFLAGS)
```

Comment:
I'm not particularly happy with the way this change is done. Perhaps we could have a `GENODE_LDFLAGS` in here in here instead:

````
COMMON_LDFLAGS=-nostdlib -z max-page-size=0x1000 $(HOST_LDFLAGS)
LDFLAGS=$(COMMON_LDFLAGS) -static
GENODE_LDFLAGS=$(COMMON_LDFLAGS)
````

Then, pass `GENODE_LDFLAGS` in the appropriate places (`.pc` and `%.genode` targets) and keep `LDFLAGS` in all the other places, rather than manually adding `-static`. What do you think?

Reply:
I don't have a problem with that, PR updated.